docs/mddocs/Quickstart/ollama_quickstart.md

```diff
@@ -183,3 +183,17 @@ An example process of interacting with model with `ollama run example` looks lik
 <a href="https://llm-assets.readthedocs.io/en/latest/_images/ollama_gguf_demo_image.png" target="_blank">
   <img src="https://llm-assets.readthedocs.io/en/latest/_images/ollama_gguf_demo_image.png" width=100%; />
 </a>
+
+### Troubleshooting
+
+#### Why model is always loaded again after several minutes
+Ollama will unload model from gpu memory in every 5 minutes as default. For latest version of ollama, you could set `OLLAMA_KEEP_ALIVE=-1` to keep the model loaded in memory. Reference issue: https://github.com/intel-analytics/ipex-llm/issues/11608
+
+#### `exit status 0xc0000135` error when executing  `ollama serve`
+If you meet `llama runner process has terminated: exit status 0xc0000135` when executing `ollama serve`, this is most likely caused by the lack of sycl dependency. Please check:

```

Comment:
When executing `ollama serve`, if you meet `llama runner process has terminated: exit status 0xc0000135` on Windows or you meet `ollama_llama_server: error while loading shared libraries: libmkl_core.so.2: cannot open shared object file` on Linux,